modules/plugin/jdbc/jdbc-h2/src/main/java/org/geotools/data/h2/H2FilterToSQL.java

```diff
@@ -217,7 +218,7 @@ protected void writeLiteral(Object literal) throws IOException {
         if (literal instanceof Date) {
             out.write("PARSEDATETIME(");
             if (literal instanceof java.sql.Date) {
-                out.write("'" + DATE_FORMAT.format(literal) + "', 'yyyy-MM-dd'");
+                out.write("'" + DATE_FORMAT.format(literal) + "', 'yyyy-MM-dd', '"+Locale.getDefault()+"', 'GMT'");
```

Comment:
Does the PARSEDATETIME function record in GMT or in the current Locale? Trying to see if the databases are portable or are specific to a timezone